1 The Physical/Guess Point Level 1 (Stateless)/1 - Red-Green-Refactor Practice/1.1 - Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -1,4 +1,8 @@
 export function fizzbuzz(number: number) {
+  if (number % 3 === 0 && number % 5 === 0) {

```

Comment:
If you wanted, you could take this one step further:
build up the result string by appending either Fizz or Buzz or both (instead of returning directly), and you can get rid of this if.

Reply:
@j92 the purpose of this exercise is to be stateless. By appending the Fizz or Buzz you would make it stateful.